demo/predict-taxi-trip-duration-nb/setup_openmldb.sh

```diff
@@ -16,7 +16,7 @@
 # limitations under the License.
 
 set -eE
-VERSION=0.3.0
+VERSION="$1"
```

Comment:
openmldb-spark should also match the version?

However, when will openmldbspark be released? Manually release after openmldb's release? If this is true, the required spark version may not exists when we are doing release for openmldb.

Reply:
@tobegit3hub openmldb-spark can be auto released?